facebookshare.module

```diff
@@ -62,108 +31,24 @@ function facebookshare_block_view($delta = '') {
 
   switch ($delta) {
     case 'facebook_share':
-      $settings = variable_get('facebookshare_block_settings', array());
-      $block['content'] = facebookshare_content($settings);
+      $current_page = $base_url . DIRECTORY_SEPARATOR . current_path();
```

Comment:
Better hard-code a slash (`/`) instead of `DIRECTORY_SEPARATOR`, since as the name states it's related to directories where it's value is subject to change from system to system. URL's slashes, on the other hand, are always the same.